src/screens/CreatePost/index.js

```diff
@@ -751,6 +752,7 @@ const CreatePost = () => {
             topicChats={listTopicChat}
             allTaggedUser={allTaggingUser}
             setAllTaggedUser={setAllTaggingUser}
+            isAnonimity={typeUser}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new prop `isAnonimity` is being passed to the component but it's not clear if this prop is being used or handled in the component. Please ensure that the component has been updated to handle this prop.
```diff
-            isAnonimity={typeUser}
+            isAnonymous={typeUser}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->